Problem with fetching OGMAcamSDK from a repo

During run_cmake process - when the script attempts to fetch the OGMAcamSDK repository, the git fetch operation fails when using commit **_5e83d81384479b1684b97e9f6936e453050da4cb_**. As a result, the run_cmake script cannot complete successfully. However, switching to the head commit **_9787aaf4882497262a8ba1612b43f93fb083d2e6_** resolves the issue, and the git fetch operation works as expected.

The error seen during build:

```
MSBuild version 17.12.12+1cce77968 for .NET Framework

  1>Checking Build System
  1>Creating directories for 'ogmacamsdk-populate'
  Performing download step (git clone) for 'ogmacamsdk-populate'
  Cloning into 'ogmacamsdk-src'...
  fatal: unable to read tree (5e83d81384479b1684b97e9f6936e453050da4cb)
  CMake Error at ogmacamsdk-subbuild/ogmacamsdk-populate-prefix/tmp/ogmacamsdk-populate-gitclone.cmake:49 (message):
    Failed to checkout tag: '5e83d81384479b1684b97e9f6936e453050da4cb'

```

--- a/thirdparty/thirdparty.cmake
+++ b/thirdparty/thirdparty.cmake
@@ -489,7 +489,6 @@ else()
     indi
     GIT_REPOSITORY https://github.com/indilib/indi.git
     GIT_TAG 856ac85b965177d23cd0c819a49fd50bdaeece60  # v2.0.5
-    GIT_SHALLOW 1
     CMAKE_ARGS -Wno-dev
       -DINDI_BUILD_SERVER=OFF
       -DINDI_BUILD_DRIVERS=OFF
@@ -601,7 +600,6 @@ if (NOT OPENSOURCE_ONLY)
     OGMAcamSDK
     GIT_REPOSITORY https://github.com/OGMAvision/OGMAcamSDK.git
     GIT_TAG 5e83d81384479b1684b97e9f6936e453050da4cb
-    GIT_SHALLOW 1
   )
   FetchContent_MakeAvailable(OGMAcamSDK)
   include_directories(${ogmacamsdk_SOURCE_DIR}/inc)

